translate/src/api/user.ts

```diff
@@ -64,6 +65,12 @@ export const fetchUserData = (): Promise<ApiUserData> => GET('/user-data/');
 /** Get all users from server. */
 export const fetchUsersList = (): Promise<UsersList[]> => GET('get-users');
 
+/** Check if the user is a new locale contributor. */
+export async function fetchIsNewContributor(locale: string): Promise<boolean> {
+  const results = await GET(`/is-new-contributor/${locale}/`);
+  return keysToCamelCase(results).isNewContributor;
```

Comment:
```suggestion
  return results.is_new_contributor;
```

Do I understand right that the only reason for the `keysToCamelCase()` call here is to be able to avoid the `_` in the API field name? That seems rather clumsy.